### DISCUSSION TITLE
GPL 3 license

### DISCUSSION THREAD

**Discussion by BradleyHill** [streetsidesoftware/cspell-dicts#605] [General]
Was the move to `"license": "GPL-3.0-or-later"` intentional. Sullying cspell-lib as now contains GPL-3 dependencies. Regards.

**Comment by Jason3S** [comment]
@BradleyHill,

Thank you for the question.

There are two different things going on:
1. `cspell-lib` license is MIT and there is no intention to transfer it to GPL-3. Because it only references libraries that are GPL-3, it does not need to become GPL-3. The `cspell-lib` is not a derivative work. See [Frequently Asked Questions about the GNU Licenses - GNU Project - Free Software Foundation](https://www.gnu.org/licenses/gpl-faq.en.html). Since the spell checker refers to the dictionaries, but does not package or bundle the dictionaries, it does not need to be GPL-3. On the other hand, [Code Spell Checker - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) does need to be GPL-3 because it bundles the dictionaries.
2.  Some of the dictionaries MUST be GPL-3 because they are derived from other GPL-3 work. Related to #441 and #440. As each dictionary is reviewed, the license is updated as appropriate.
